README.md

```diff
@@ -1,5 +1,122 @@
 # Drone3D
+[![Build Status](https://api.cirrus-ci.com/github/Drone3D-Team/Drone3D.svg)](https://cirrus-ci.com/github/Drone3D-Team/Drone3D)
+[![Maintainability](https://api.codeclimate.com/v1/badges/7d12c4fd472569490788/maintainability)](https://codeclimate.com/github/Drone3D-Team/Drone3D/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/7d12c4fd472569490788/test_coverage)](https://codeclimate.com/github/Drone3D-Team/Drone3D/test_coverage)
+
+
+## Table of contents
+* [Description](#description)
+* [Usage and features](#usage-and-features)
+* [Set up](#set-up)
+* [Developement](#developement)
+* [License](#license)
+
+## Description
+Drone3D is the mobile application that let you plan and execute drone mission to take pictures of a location. These photos can then be used to reconstruct a 3D model of the area. You have the possibility to share the missions you've planned with other or see their missions. This app can be used with any drone or simulation supporting [MAVSDK](https://mavsdk.mavlink.io/main/en/index.html).
+
+## Usage and features
+
+### Register and login
+TODO add screen login / register
+
+You can create a account or login in a previously created account with a email and a password. Once you are logged-in you'll be able to store and share missions.
+
+### Connect a drone or a simulation
+TODO add screen connect a drone
+
+To connect a drone you'll have power up your drone set it up to communicate through the udp://:14540 port, then click on the "connect a drone" button, it will open the port and search for the drone for three seconds before timing out. Repeat the operation as many time as needed.
+To connect a simulation, you'll have first to create to launch a simulation with a tool such as [auterion](https://suite.auterion.com/login?returnUrl=%2Fsimulations) and then enter the corresponding the IP and port, click on the "connect a simulation" button, it can take a few seconds.
+
+### Manage offline map
+TODO add screen offline map
+
+This page lets you download a part of the map. Select the on the map the area you want to save by zooming on the map. Once you have it on your screen, press the save button, give it a name and download it, depending on the size of the area it can take a few seconds. You have space to download up to 6000 tiles, the number of tiles can vary based on the size of the area and the precison of the map. If you no longer need a previously saved area and want to free some space, you can delete it by pressing the corresponding delete button.
```

Comment:
You probably wanted to write
```suggestion
This page lets you download a part of the map. Select the area you want to save by zooming on the map. Once you have it on your screen, press the save button, give it a name and download it, depending on the size of the area it can take a few seconds. You have space to download up to 6000 tiles, the number of tiles can vary based on the size of the area and the precision of the map. If you no longer need a previously saved area and want to free some space, you can delete it by pressing the corresponding delete button.
```